src/lib/core/CHIPConfig.h

```diff
@@ -2498,6 +2498,41 @@ extern const char CHIP_NON_PRODUCTION_MARKER[];
 #define CHIP_CONFIG_VERBOSE_VERIFY_OR_DIE 0
 #endif
 
+/**
+ * @def CHIP_CONFIG_MAX_GROUPS_PER_FABRIC
+ *
+ * @brief Defines the number of groups supported per fabric, see Group Key Management Cluster in specification.
+ *
+ * Binds to number of GroupState entries to support per fabric
+ */
+#ifndef CHIP_CONFIG_MAX_GROUPS_PER_FABRIC
+#define CHIP_CONFIG_MAX_GROUPS_PER_FABRIC 5
+#endif
+
+// TODO: Need to cap number of KeySets
+
+/**
+ * @def CHIP_CONFIG_MAX_GROUP_ENDPOINTS_PER_FABRIC
+ *
+ * @brief Defines the number of "endpoint->controlling group" mappings per fabric.
+ *
+ * Binds to number of GroupMapping entries per fabric
+ */
+#ifndef CHIP_CONFIG_MAX_GROUP_ENDPOINTS_PER_FABRIC
+#define CHIP_CONFIG_MAX_GROUP_ENDPOINTS_PER_FABRIC 8
+#endif
```

Comment:
Since the unit test already handles setting the values needed for tests, suggest:

1- `CHIP_CONFIG_MAX_GROUPS_PER_FABRIC` == 1
2- `CHIP_CONFIG_MAX_GROUP_ENDPOINTS_PER_FABRIC` == 1

That would reduce need for memory for default cases where groups are not currently used.

We should also have a different constant that is for keysets per fabric, that should be 1+1 (IPK + 1 group key sert used for all groups on a given fabric) until the spec has a minimum-to-support number of keyset for when groups are used (which right now is 1: the IPK).

Reply:
Defaults set to 1